lightning/src/ln/chan_utils.rs

```diff
@@ -279,9 +277,9 @@ pub struct ChannelPublicKeys {
 	/// a commitment transaction so that their counterparty can claim all available funds if they
 	/// broadcast an old state.
 	pub revocation_basepoint: PublicKey,
-	/// The base point which is used (with derive_public_key) to derive a per-commitment payment
-	/// public key which receives immediately-spendable non-HTLC-encumbered funds.
-	pub payment_basepoint: PublicKey,
+	/// The public key which receives immediately-spendable non-HTLC-encumbered funds. This key is
```

Comment:
non-HTLC-encumbered funds is a bit unclear, specially after anchor. Maybe our "balance" output ? Aalso "across every remote commitment"